src/main/java/core/org/akaza/openclinica/bean/extract/odm/ClinicalDataReportBean.java

```diff
@@ -344,9 +370,25 @@ public void addNodeClinicalData(boolean header, boolean footer) {
                                 if (updatedBy != null && updatedBy.length() > 0) {
                                     xml.append("\" OpenClinica:UpdatedBy=\"" + StringEscapeUtils.escapeXml(updatedBy));
                                 }
-                                String status = form.getStatus();
-                                if (status != null && status.length() > 0) {
-                                    xml.append("\" OpenClinica:Status=\"" + StringEscapeUtils.escapeXml(status));
+
+                                if(form.getWorkflowStatus()!=null) {
+                                    EventCrfWorkflowEnum workflow = form.getWorkflowStatus();
+                                    if (!StringUtils.isEmpty(workflow)) {
+                                        xml.append("\" OpenClinica:Status=\"" + StringEscapeUtils.escapeXml( workflow.getDisplayValue()));
```

Comment:
For backwards compatibility reasons, we should continue to set the status to removed, archived, etc. based on the new fields along with the new fields you added below.  It is okay if you want to handle backwards compatibility in a separate PR.

Reply:
In the code ,I  have included the logic for backwards compatibility. first, liquibase will covert data from Status column into new columns workflow_status, removed, archived and locked. The only place where data is not converted is the audit log table. I added separate logic in UI for existing data in audit log event page.

Reply:
I am talking more about what gets included in the ODM. We want to make sure that "OpenClinica:Status" can still show values like removed, archived, etc for backwards compatibility reasons so that the existing integrations that rely on the "OpenClinica:Status" field will continue to work.

Reply:
Got you,